pylsp_jsonrpc/__init__.py

```diff
@@ -1,6 +1,21 @@
 # Copyright 2017-2020 Palantir Technologies, Inc.
 # Copyright 2021- Python Language Server Contributors.
 
+from . import _version
 from ._version import __version__
 
-__all__ = [__version__]
+def convertVersionInfo(version: str) -> (int, ..., str):
```

Comment:
```suggestion
def convert_version_info(version: str) -> (int, ..., str):
```

Change from camel-case to snake-case because that's our convention.